CHANGES.rst

```diff
@@ -18,7 +18,7 @@ background
 ----------
 
 - Update the background subtraction step to accept rateints (3D) input
-  background exposures. [#7049]
+  background exposures. [#7049] Updated docs accordingly. [#7055]
```

Comment:
Can you modify this so that all the text is together, and the two PR's can be listed as `[#7049, #7055]`.